src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
Classes should not begin with an empty line after the class declaration. Please remove the empty line to adhere to Java best practices.